internal/validators/status/validator_test.go

```diff
@@ -317,6 +317,91 @@ func Test_statusValidator_listStatues(t *testing.T) {
 		want    []*ghaStatus
 	}
 	tests := map[string]test{
+		"succeeds to get job statuses even if the same job exists": func() test {
+			c := &mock.Client{
+				GetCombinedStatusFunc: func(ctx context.Context, owner, repo, ref string, opts *github.ListOptions) (*github.CombinedStatus, *github.Response, error) {
+					return &github.CombinedStatus{
+						Statuses: []*github.RepoStatus{
+							{
+								Context: stringPtr("job-01"),
+								State:   stringPtr(successState),
+							},
+							{
+								Context: stringPtr("job-01"), // Same as previous job name.
```

Comment:
Just to understand, in this test case, is this the "previous job run", or are you just saying that this is the same name as the 0th element in this slice? Which one is considered to be the latest state?

Reply:
The first element is latest state. (0th element in this slice). 
I checked the github action api resoonses and it was added in the latest order.